Doppler.ContactPolicies.Api/Controllers/ContactPolicies.cs

```diff
@@ -13,11 +14,21 @@ namespace Doppler.ContactPolicies.Api.Controllers
     [ApiController]
     public class ContactPolicies : ControllerBase
     {
-        [Authorize(DopplerSecurity.Policies.OWN_RESOURCE_OR_SUPERUSER)]
+        private readonly IContactPoliciesService _contactPoliciesService;
+
+        public ContactPolicies(IContactPoliciesService contactPoliciesService)
+        {
+            _contactPoliciesService = contactPoliciesService;
+        }
+
+        [Authorize(Policies.OWN_RESOURCE_OR_SUPERUSER)]
         [HttpGet("/accounts/{accountName}/settings")]
-        public string GetContactPoliciesSettings(string accountName)
+        public async Task<IActionResult> GetContactPoliciesSettings(string accountName)
         {
-            return $"Hello! \"you\" that have access to the account with accountName '{accountName}'";
+            var contactPoliciesSettings = await _contactPoliciesService.GetContactPoliciesSettingsAsync(accountName);
```

Comment:
```suggestion
            var contactPoliciesSettings = await _contactPoliciesService.GetContactPoliciesSettingsAsync(accountName);
            
            if (contactPoliciesSettings == null)
            {
              return BadRequest($"No contact policies settings for account name: {accountName}");
            }
```

Reply:
great thanks @ms-darianbenito

Reply:
Mmm, I do not agree. 

We should understand all the possible outputs for these scenarios:

1. User does not exist
2. User exists but does not have an entry in `UserShippingLimit`
3. User exists, has an entry in `UserShippingLimit`, but `UserShippingLimit.Enabled = false OR UserShippingLimit.Enabled IS NULL` 
4. User exists, has an entry in `UserShippingLimit`, and `UserShippingLimit.Enabled = true`

In my opinion, for the GET, we do not need the result of permissions API, so, I think that these could be the output in each case:

1. User does not exist

  * Return `404 NOT FOUND` error

2. User exists but does not have an entry in `UserShippingLimit`

  * Return an empty not active result. For example:

    ```json
    {
      active: false,
      emailsAmountByInterval: null,
      intervalInDays: null,
      subscribersLists: []
    }
    ```

    or better yet:


    ```json
    {
      active: false
    }
    ```

3. User exists, has an entry in `UserShippingLimit`, but `UserShippingLimit.Enabled = false OR UserShippingLimit.Enabled IS NULL` 

    * Return an empty not active result (the same that in point 2)

4. User exists, has an entry in `UserShippingLimit`, and `UserShippingLimit.Enabled = true`

    * A complete object with the expected data, for example:

    ```json
    {
      active: true,
      emailsAmountByInterval: 10,
      intervalInDays: 2,
      subscribersLists: [ 1234, 567, 98765 ]
    }
    ```

Reply:
BTW, if I am not wrong, these cases are not being tested. Could you add tests for them?